recipes/weldx/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "weldx" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  #git_url: /home/marscher/sources/weldx
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 707fa793d92089e976865870dc603ce9cf34cea4c9af66fd6aff9a6c7c3c79d1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
I've noticed, that CF updated the documentation to use this command. But I do not see, why dragging in external dependencies via pip would be helpful in any way. Is there a new logic inside of conda-build to prevent this situation? If not, I'd like to stick to the old behavior of failing, if some deps are not declared in meta.yaml.

Reply:
Conda build now handles communicating this to pip automatically with environment variables which makes it much harder to get wrong: https://gitter.im/conda-forge/conda-forge.github.io?at=60368c3fb18370540514791b

Reply:
Thanks a lot for the explanation! That's a really nice improvement to conda-bld.